examples/resources/images/license.md

```diff
@@ -102,6 +102,14 @@ https://opengameart.org/content/super-mario-world-style-tiles
 CC0 1.0
 ```
 
+## blend/dest.png
```

Comment:
Put this in between `audio/pause.png` and `flappy/gopher.png` for alphabetical order